graylog2-web-interface/src/components/indices/IndexMaintenanceStrategiesSummary.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Alert } from 'react-bootstrap';
+import Spinner from 'components/common/Spinner';
+
+const IndexMaintenanceStrategiesSummary = React.createClass({
+  propTypes: {
+    config: React.PropTypes.object.isRequired,
+  },
+
+  render() {
+    if (!this.props.config) {
+      return (<Spinner />);
+    }
+
+    const activeStrategy = this.props.config.strategy;
+    const strategy = this.props.pluginExports.find((strategy) => strategy.type === activeStrategy);
```

Comment:
`pluginExports` is missing in `propTypes`, also `strategy` is declared twice, in different scopes.